tests/josegonzalez/Dotenv/LoaderTest.php

```diff
@@ -141,6 +141,7 @@ public function testParse()
             'STRING_FALSE' => 'false',
             'STRING_EMPTY' => '',
             'STRING_EMPTY_2' => '',
+            '01SKIPPED'=>'skipped',
```

Comment:
Yeah it will do when I start building the strict parser for the next milestone. But for now it'll be fine